src/components/MainNav.js

```diff
@@ -42,7 +42,7 @@ export default function SimpleBottomNavigation() {
     >
       <BottomNavigationAction
         style={{ color: "white" }}
```

Comment:
you can make a classname and then implement this style

Reply:
I will change this.